build/ci/linux/setup.sh

```diff
@@ -136,12 +137,13 @@ cmake --version
 # POST INSTALL
 ##########################################################################
 
-chmod +x "${ENV_FILE}"
+chmod +x "$ENV_FILE"
```

Comment:
I believe the curly braces should rather stay, for it to not evailuate `$ENV` and append `"_FILE"`to it?

Reply:
> I believe the curly braces should rather stay, for it to not evailuate `$ENV` and append `"_FILE"`to it?

I agree, I will return it

Reply:
returned